plugins/item_licenses/plugin_tests/itemLicensesSpec.js

```diff
@@ -0,0 +1,425 @@
+/* globals girderTest, describe, expect, it, runs, waitsFor, _prepareTestUpload  */
+
+girderTest.addCoveredScripts([
+    '/static/built/plugins/item_licenses/templates.js',
+    '/plugins/item_licenses/web_client/js/setup.js',
+    '/plugins/item_licenses/web_client/js/ConfigView.js',
+    '/plugins/item_licenses/web_client/js/ItemWidget.js',
+    '/plugins/item_licenses/web_client/js/SelectLicenseWidget.js'
+]);
+
+girderTest.importStylesheet(
+    '/static/built/plugins/item_licenses/plugin.min.css'
+);
+
+girderTest.startApp();
+
+describe('item_licenses plugin test', function () {
+    it('registers admin user',
+        girderTest.createUser('admin',
+                              'admin@email.com',
+                              'Admin',
+                              'User',
+                              'CxV`%EOsq9'));
+
+    it('logs out', girderTest.logout());
+
+    it('registers normal user',
+        girderTest.createUser('user',
+                              'user@email.com',
+                              'Normal',
+                              'User',
+                              'a*z/Swb?td'));
+
+    it('creates an item with a license', function () {
+        waitsFor(function () {
+            return $('a.g-my-folders').length > 0;
+        }, 'my folders link to load');
+        runs(function () {
+            $('a.g-my-folders').click();
+        });
+        girderTest.waitForLoad();
+
+        waitsFor(function () {
+            return $('a.g-folder-list-link:contains(Public)').length > 0;
+        }, 'the folders list to load');
+        runs(function () {
+            $('a.g-folder-list-link:contains(Public)').click();
+        });
+        girderTest.waitForLoad();
+
+        waitsFor(function () {
+            return $('button.g-folder-actions-button:visible').length === 1;
+        }, 'folder actions button to be visible');
+        runs(function () {
+            $('button.g-folder-actions-button:visible').click();
+        });
+        girderTest.waitForLoad();
+
+        waitsFor(function () {
+            return $('a.g-create-item:visible').length === 1;
+        }, 'create item here link to be visible');
+        runs(function () {
+            $('a.g-create-item:visible').click();
+        });
+        girderTest.waitForLoad();
+
+        waitsFor(function () {
+            return Backbone.history.fragment.slice(-18) === '?dialog=itemcreate';
+        }, 'the URL state to change');
+        waitsFor(function () {
+            return $('button.g-save-item:visible').text() === 'Create';
+        }, 'the create item dialog to appear');
```

Comment:
The `girderTest.waitForLoad` on line 65 is incorrect, and yes there should be a `girderTest.waitForDialog` after waiting for the URL state change.

That said, are you recommending replacing both of these waitFor() blocks with `girderTest.waitForDialog`? Currently, after the changes mentioned above, this is very similar to what's done here: https://github.com/girder/girder/blob/f6f259e3e6c999542cc5e134da1200eb4b71ff21/clients/web/test/spec/itemSpec.js#L28-L39